src/test/resources/schema.sql

```diff
@@ -6,66 +6,72 @@ begin;
 drop table if exists dataset;
 drop table if exists record_log;
 drop table if exists record_error_log;
+drop table if exists record;
 drop table if exists thumbnail;
-drop table if exists transform_xslt;
+drop table if exists default_transform_xslt;
 commit;
 
 begin;
-
 create table if not exists dataset (
-   dataset_id int generated by default as identity,
-   dataset_name varchar(255) not null,
-   records_quantity integer not null,
-   created_date timestamp with time zone default now(),
-   country varchar(35),
-   language varchar(3),
-   record_limit_exceeded bool,
-   xslt_edm_external_content text,
-   Primary Key (dataset_id)
-);
+    dataset_id serial,
+    dataset_name varchar(255) not null,
+    records_quantity integer not null,
+    created_date timestamp with time zone default now(),
+    country varchar(35) not null,
+    language varchar(3) not null,
+    record_limit_exceeded bool,
+    xslt_edm_external_content text,
+    Primary Key (dataset_id)
+    );
 
-create table if not exists record_log
+create table if not exists record
 (
-    id           int generated by default as identity,
-    record_id    varchar(100) not null,
-    dataset_id   varchar(100) not null,
-    step         varchar(30)  not null,
-    status       varchar(30)  not null,
-    content      text         not null,
+    id bigserial,
+    europeana_id varchar(100) null,
+    provider_id varchar(100) null,
+    dataset_id varchar(100) not null,
+    content text not null,
```

Comment:
Hi @JoanaCMS , I just happened to see this, so I thought I'd ask the question. The content here is kept on a record level, but not on a step level. Are you sure that is correct? Shouldn't you keep a version of the record content for each step?

Reply:
When I first did this change, I focused on saving the data on a record level, since this table is a representation of a record and its details. But I see your point, we would want to save the content of the record for each step to check how it is, which would be useful specially for debugging. You are right, we should add a new column `step` in it to save the status of the content in the record.